src/components/manage/Contents/Contents.jsx

```diff
@@ -1150,16 +1162,17 @@ class Contents extends Component {
                                 'EffectiveDate',
                                 'CreationDate',
                                 'ModificationDate',
-                                'Type',
+                                'portal_type',
                               ],
                               index => (
-                                <Dropdown.Item key={index}>
+                                <Dropdown.Item key={index} data-cy={index}>
```

Comment:
@ksuess this might be fragile. An index is a number that is subject to change if the items in the dropdown change.

Reply:
index iterates 
                              [
                                'id',
                                'sortable_title',
                                'EffectiveDate',
                                'CreationDate',
                                'ModificationDate',
                                'portal_type',
                              ]